@xen-orchestra/fs/src/abstract.js

```diff
@@ -223,6 +223,10 @@ export default class RemoteHandlerAbstract {
     await this._forget()
   }
 
+  async getInfo(): Promise<Object> {
```

Comment:
This is a bit vague, can you add type for each properties (that we use and test) that can be in the object?